mm2src/mm2_bitcoin/chain/src/block_header.rs

```diff
@@ -92,6 +93,8 @@ pub struct BlockHeader {
     pub version: u32,
     pub previous_header_hash: H256,
     pub merkle_root_hash: H256,
+    /// https://github.com/lbryio/lbrycrd/blob/71fc94b1dea1c7818f84a09832ec3db736481e0f/src/primitives/block.h#L40
+    pub claim_trie_root: Option<H256>,
     pub hash_final_sapling_root: Option<H256>,
```

Comment:
Fields naming should be the same as in related coin's codebase. I guess LBC doesn't support Sapling, so using `hash_final_sapling_root` for its block header field will be confusing.

Reply:
> I guess this might affect the coins deserializing in the case of `LBC` and `Zcash` as I think the field name should correspond with the incoming fields..maybe I'm wrong! @shamardy

Deserializing in this case is unrelated to the field names since you deserialize bytes not JSON.

> Fields naming should be the same as in related coin's codebase. I guess LBC doesn't support Sapling, so using `hash_final_sapling_root` for its block header field will be confusing.

I didn't mean to use `hash_final_sapling_root` for LBC but to choose a name that represents both coins. But I guess we can just leave it as is for now and if there are other coins in the future that have similar `trie_root` field in the same place we can combine the `claim_trie_root` of LBC with it.